src/webgpu/api/operation/rendering/depth.spec.ts

```diff
@@ -3,13 +3,211 @@ Test related to depth buffer, depth op, compare func, etc.
 `;
 
 import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { TypedArrayBufferView } from '../../../../common/util/util.js';
 import { kDepthStencilFormats, kTextureFormatInfo } from '../../../capability_info.js';
 import { GPUTest } from '../../../gpu_test.js';
 
+class DepthTest extends GPUTest {
+  checkStencilCompareFunction(
```

Comment:
Stencil is mostly a separate feature from depth, so perhaps put this in a new file. There are some interactions, like `depthFailOp`, but those would probably fit best into the stencil file.

Reply:
Ah, I see. I create stencil.spec.ts file, and then move this test to the file.